portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,44 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Chooses a random project and updates the following: 
+ * img element with the project's img URl 
+ * heading element with the project's name
+ * paragraph element with the project's description
+ * link element with the project's Github link
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomProject() {
+  const projects =
+      [
+          { name:"Tak", 
+            text:"This is terminal version of the KingKiller Chronicle's two player game with the same name!", 
+            link:"https://github.com/connielei-stuy/apcs1-final-tak", 
+            image:"/images/tak.png"
+          }, 
+          { name:"TriWizard Maze", 
+            text:"This is virtual reality experience of the TriWizard Maze!", 
+            link:"https://github.com/connielei-kwk/kwk17-FINAL-PROJ-maze", 
+            image:"/images/tri.png"
+          },
+          { name:"Graphics Engine", 
+            text:"This is simple graphics engine that can create basic shapes.", 
+            link:"https://github.com/connielei-stuy/graphics-anim", 
+            image:"/images/graphics.gif"
+          } 
+      ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const project = projects[Math.floor(Math.random() * projects.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const projectName = document.getElementById('project-name');
+  projectName.innerText = project['name'];
+
+  const projectTextContainer = document.getElementById('project-text');
+  projectTextContainer.innerText = project['text'];
+
+  const projectLink = document.getElementById('project-link');
+  projectLink.innerText = "Click here for the repo!";
+  projectLink.href = project['link'];
+
+  const projectImg = document.getElementById('project-img');
```

Comment:
Consider writing sub methods for these. e.g.

```
updateProjectName(name) {
  projectTextContainer.innerText = name
}
```

this may seem like an overkill, but this will encapsulate the logic for each piece inside smaller methods. Making your code easier to follow. e.g. this could end up looking like:

```
function addRandomProject() {
  const project = getRandomProject();

  updateProjectName(project.name);
  updateProjectText(project.text);
  updateProjectLink(project.link);
}
```